src/sql/src/plan/statement/show.rs

```diff
@@ -1035,17 +1037,34 @@ fn humanize_sql_for_show_create(
                         }
                     });
                 }
-                CreateSourceConnection::Kafka { .. }
-                | CreateSourceConnection::LoadGenerator { .. } => {}
+
+                CreateSourceConnection::LoadGenerator { .. } => {
+                    // Multi-output load generator sources only support `FOR ALL
+                    // TABLES`.
+                    if !curr_references.is_empty() {
+                        curr_references.clear();
+                        stmt.referenced_subsources = Some(ReferencedSubsources::All);
+                    }
```

Comment:
Possibly untested (if coverage data is accurate).

Reply:
Ah, this would be triggered by `SHOW CREATE SOURCE` on a load generator source. I can add this test.